doc/package-notes.xml

```diff
@@ -654,6 +654,68 @@ overrides = self: super: rec {
 </screen>
   </section>
  </section>
+ <section xml:id="sec-firefox">
+  <title>Firefox</title>
+
+  <section xml:id="sec-firefox-config">
+   <title>Configuring the Firefox Wrapper</title>
+
+   <para>
+    The firefox (wrapper) package can be configured through several mechanisms.
+    The older
+    <link xlink:href="https://support.mozilla.org/en-US/kb/customizing-firefox-using-autoconfig">
+    AutoConfig </link> method, and the more recent
+    <link xlink:href="https://github.com/mozilla/policy-templates/blob/master/README.md#extensions">
+    enterprise policies</link> corresponding to the <literal> extraPrefs
+    </literal> and <literal> extraPolicies </literal> arguments of the firefox
+    wrapper package.
+   </para>
+
+   <para>
+    For example the option <literal>media.autoplay.default=1</literal> which
+    disables autoplaying videos, could be specified as follows (the syntax of
+    AutoConfig is a superset of user.js):
+<programlisting>
+firefox.override { extraPrefs = '' pref("media.autoplay.default",1); ''; }
+</programlisting>
+    To disable upload of telemetry data, you could use:
+<programlisting>
+firefox.override { extraPolicies = { DisableTelemetry = true; }; }
```

Comment:
I don't like that the json options start with an uppercase letter, going against the conventional nix style. I don't really know what to do about it though.